cdap-ui/app/cdap/components/DataPrep/TopPanel/PipelineConfigHelper.js

```diff
@@ -344,7 +344,7 @@ function constructSpannerSource(artifactsList, spannerInfo) {
     return T.translate(`${PREFIX}.spanner`);
   }
 
-  batchArtifact.version = '[0.9.2, 3.0.0)';
+  batchArtifact.version = '[0.11.0-SNAPSHOT, 3.0.0)';
```

Comment:
I meant like - this is the version for which artifact?

Reply:
It's the `google-cloud` plugins artifact

Reply:
Renamed variable to `googleCloudArtifact`